pkg/components/render_test.go

```diff
@@ -64,57 +64,6 @@ func Test_renderLvmdParams(t *testing.T) {
 	}
 }
 
-func Test_renderLvmdConfig(t *testing.T) {
```

Comment:
I removed it because it was testing if the ConfigMap is properly templated (`{{ .lvmd }}`) but now it's substituted so this test didn't made much sense:
```
# pkg/components/storage.go
-	if err := assets.ApplyConfigMaps(cm, renderTemplate, lvmdRenderParams, kubeconfigPath); err != nil {
+	if err := assets.ApplyConfigMapWithData(cm, map[string]string{"lvmd.yaml": lvmdRenderParams["lvmd"].(string)}, kubeconfigPath); err != nil {
```

I think, if we'd want to test something, it would be `assets.ApplyConfigMapWithData()` (although I think it would need a refactor to split this logic from applying it to API)

Reply:
OK, it makes sense to remove this. We can add other tests in separate PRs if we want them.